### ISSUE TITLE
Update Android instructions for using correct AVD

### ISSUE THREAD

**Issue by lorensr** [meteor/meteor#5380]
@martijnwalraven 😊

After completing

https://github.com/meteor/meteor/wiki/Mobile-Development-Install:-Android-on-Mac#installing-android-studio-and-completing-the-setup-wizard

when I did `meteor run android`, a API v23 emulator appeared, which is the wrong one for Cordova. `android avd` showed:

![image](https://cloud.githubusercontent.com/assets/251288/10328415/46aa3ffe-6c82-11e5-901c-b6abedd5f973.png)

I was unable to create a new one:

![image](https://cloud.githubusercontent.com/assets/251288/10329431/5f05d506-6c8f-11e5-903d-0e59ac8aec1e.png)

So I went back to the SDK Manager and downloaded v22 system images for x86 and arm (which the wiki does not say to do), and then restarted `android avd` and created a new AVD. 

# Suggestions

1- Update the wiki explaining how to do the above

2- The SDK screenshot is out of date – a new version of the Studio was released with a different UI. Clicking "Configure -> SDK Manager" now shows this:

![image](https://cloud.githubusercontent.com/assets/251288/10329453/c84fb978-6c8f-11e5-945d-574dcfc9b116.png)

3- Document in `meteor help run android` how to choose which AVD gets launched.

**Comment by martijnwalraven** [comment]
Thanks for reporting this. I haven't had a chance to try Android Studio 1.4, but the instructions should definitely be updated. (And they will have to be changed again when we integrate a version of Cordova Android that requires API 23!).

There is currently no way to choose your own AVD with `meteor run android`, but we should add this (also the ability to choose a target for iOS).

**Comment by pors** [comment]
See also https://github.com/meteor/meteor/issues/5358

**Comment by lorensr** [comment]
Yeah, it took a lot of random fiddling with v22 AVD settings before I found one that finished booting.

**Comment by wsjwong** [comment]
I have downloaded the API22 but the AVD manager cannot find my targets (API). So its target is disabled. Any ideas on how to let the AVD manager locate the target correctly?

**Comment by iboxgithub** [comment]
Issue solved by creating my own AVD "testAVD"
After that, when you launch 'meteor run android' you can sse in the log that Android doesn't find any default AVD and so select yours.
````
It would be interesting to update the tutorial
````

**Comment by aprevost** [comment]
I'm brand new to Meteor, just installed it and am working my way through the todo-list turorial.

I am also brand new to Android development, so of course I downloaded the latest and greatest version of Android Studio for Mac OSX. That means following the instructions on the page "meteor install-sdk android" now sends you to (https://github.com/meteor/meteor/wiki/Mobile-Development-Install:-Android-on-Mac) did not work for me. It took me a lot of poking around to find the issues here with the info I needed to get the emulator working.

I think most people who are new to meteor will also be installing latest stable versions of everything. So really should add to the documentation on that page (and/or possibly the tutorial) ASAP. Need to include the extra steps of running "android avd" and manually creating an AVD with the exact specifications outlined by @lorensr above, or other newcomers to the platform will waste a lot of time trying to make this work.
